test/common-utils/config-subdirectory.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Definition specific tests
+check "owned config directory" ls -ld ~/.config/subdirectory | awk '{print $3}' | grep 'devcontainer'
+check "owned config directory" ls -ld ~/.config | awk '{print $3}' | grep 'devcontainer'
```

Comment:
```suggestion
check "owned-config-sub-directory" bash -c "ls -ld ~/.config/subdirectory | awk '{print $3}' | grep 'devcontainer'"
check "owned-config-directory" bash -c "ls -ld ~/.config | awk '{print $3}' | grep 'devcontainer'"
```

The tests should run if you pass it within `bash -c`

Reply:
I've updated as suggested, however I'm not sure why it should make a difference, given that it runs fine locally (and I'm assuming the script runs _inside_ the container), and there are other scripts that do not use `bash -c`

Reply:
When I tested it locally, it did not run checks for me. Also, we need to wrap within `bash -c` as these commands are piped, looks like `check` function doesn't handle that well.

Reply:
The tests pass! 🎉